xgboost_synthetic/build-train-deploy.ipynb

```diff
@@ -27,22 +27,7 @@
    "cell_type": "markdown",
```

Comment:
<p>When I tried this notebook, seems <code>retrying</code> doesn't need to be installed if you install <code>requirements.txt</code>.</p><p> </p><p>Could you verify if we can remove it? Otherwise, we should add it to requirements.txt.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Seems retrying imported before requirements.txt install